packages/flutter/lib/src/material/slider.dart

```diff
@@ -684,26 +684,41 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
   void _actionHandler(_AdjustSliderIntent intent) {
     final _RenderSlider renderSlider = _renderObjectKey.currentContext!.findRenderObject()! as _RenderSlider;
     final TextDirection textDirection = Directionality.of(_renderObjectKey.currentContext!);
+    bool sliderIncreased = false;
+    widget.onChangeStart?.call(widget.value);
+
     switch (intent.type) {
       case _SliderAdjustmentType.right:
         switch (textDirection) {
           case TextDirection.rtl:
             renderSlider.decreaseAction();
           case TextDirection.ltr:
             renderSlider.increaseAction();
+            sliderIncreased = true;
         }
       case _SliderAdjustmentType.left:
         switch (textDirection) {
           case TextDirection.rtl:
             renderSlider.increaseAction();
+            sliderIncreased = true;
           case TextDirection.ltr:
             renderSlider.decreaseAction();
         }
       case _SliderAdjustmentType.up:
         renderSlider.increaseAction();
+        sliderIncreased = true;
       case _SliderAdjustmentType.down:
         renderSlider.decreaseAction();
     }
+
+    if(sliderIncreased)
+    {
+      widget.onChangeEnd?.call(_lerp(clampDouble(renderSlider.value  + renderSlider._semanticActionUnit,0.0,1.0)));
+    }
```

Comment:
Question:  Do you know at what point in the process this code goes through style checking?  I notice two spaces in front of the + on line 716, which I'd expect something to catch and/or correct.

Reply:
I think It goes thru a linter when you push to the main Flutter repo. I'll look at the style guide again.